README.adoc

```diff
@@ -576,3 +509,76 @@ By default James is configured without LDAP support.
 == Know more about James design
 
 James comes with a https://james.apache.org/documentation.html[Documentation] and https://github.com/linagora/james-project/tree/master/src/adr[Architectural Decision Records].
+
+== Articles for James community
+
+=== How to build and publish the website
```

Comment:
I propose to be moving these files to `/docs/modules/community/pages/website.md` and add a link to it in the README.

Reply:
I think it's a good plan to move sections of little interest away from readme. I created website.adoc (not website.md to keep the pattern of other files) and moved content there. I added a direct link to it from readme. If I understand correctly then once the antora documentation is ready and published the link will somewhat change.